tests/components/enphase_envoy/test_select.py

```diff
@@ -71,106 +68,95 @@ async def test_no_select(
 @pytest.mark.parametrize(
     ("mock_envoy"), ["envoy_metered_batt_relay"], indirect=["mock_envoy"]
```

Comment:
When there's only one param, you don't need to wrap it in a tuple and you can set `indirect=True`:

```suggestion
    "mock_envoy", ["envoy_metered_batt_relay"], indirect=True
```